src/lib_sdl/video.cr

```diff
@@ -95,6 +100,31 @@ lib LibSDL
     RESET_ISOLATION_FLAG    = 0x0008
   end
 
+  struct SDL_Version
+    major : UInt8
+    minor : UInt8
+    patch : UInt8
+  end
+
+  enum SDL_SYSType
+    SDL_SYSWM_UNKNOWN
+    SDL_SYSWM_WINDOWS
+    SDL_SYSWM_X11
+    SDL_SYSWM_DIRECTFB
+    SDL_SYSWM_COCOA
+  end
+
+  struct WM_x11
+    display : Void*
+    window : UInt64
```

Comment:
The actual type chain in C appears to be `Window` -> `XID` -> `CARD32` -> unsigned 32-bit integer.